src/scripts/cache-save.sh

```diff
@@ -63,7 +63,7 @@ fi
 
 LOCKFILE_PATH="${CACHE_DIR}/lockfile"
 
-if [ -f "${LOCKFILE_PATH}" ]; then
+if [ -L "${LOCKFILE_PATH}" ]; then
     unlink "${LOCKFILE_PATH}"
```

Comment:
Another solution would be to simply `rm -f "${LOCKFILE_PATH}"` without testing if it exists and/or is a symlink.

Reply:
Yes, this is also a possibility!

Reply:
@clementlecorre Triggered the integration tests here, and it seems as if this still fails. I believe that `rm -f` might be the best case. @clementlecorre Would you like to update the pr, or shall I? 

https://app.circleci.com/pipelines/github/CircleCI-Public/python-orb/580/workflows/64d7837d-4817-48c4-bb4d-0c5af70c5cfa/jobs/3525?invite=true#step-107-76

Reply:
I'm doing it!

Reply:
@clementlecorre Thank you! Tests passed. Could you verify this resolves your issue by targeting the dev:alpha version? (`python@dev:alpha`)